tests/vmi_multus_test.go

```diff
@@ -499,8 +499,8 @@ var _ = Describe("[Serial]Multus", func() {
 					linuxBridgeNetwork,
 				}
 
-				ep1Ip := "1.0.0.10/24"
```

Comment:
Same here, you cannot drop the CIDR. It is used for the `ip link add` command.

Reply:
I should be more careful while checking tests, thanks for catching this.

Reply:
Test is adjusted now, let's see how it goes.